.github/workflows/test.yml

```diff
@@ -19,10 +19,18 @@ jobs:
         uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node-version }}
-      - name: build and test
+      - name: build
         run: |
           yarn --ignore-engines
           yarn build
-          yarn test
+        env:
+          CI: true
+      - name: test
+        run: npm test
+        env:
+          CI: true
```

Comment:
I had to debug the log outputs a lot while working on this pull request. It helped me to seperate the build steps from the testing, much less noise I had to scroll through